src/nunit.analyzers.tests/TestCaseUsage/TestCaseUsageAnalyzerTests.cs

```diff
@@ -15,6 +15,16 @@ namespace NUnit.Analyzers.Tests.TestCaseUsage
     public sealed class TestCaseUsageAnalyzerTests
     {
         private DiagnosticAnalyzer analyzer = new TestCaseUsageAnalyzer();
+        
+        private static IEnumerable<TestCaseData> SpecialConversions
+        {
+            get
+            {
+                yield return new TestCaseData("2019-10-10", typeof(DateTime));
+                yield return new TestCaseData("23:59:59", typeof(TimeSpan));
+                yield return new TestCaseData("2019-10-10", typeof(DateTimeOffset));
```

Comment:
Can you also provide a testcase where we have time and offset for the `DateTimeOffset`?